features.json

```diff
@@ -124,6 +124,22 @@
 				"simd": "--enable-simd",
 				"threads": "--enable-threads"
 			}
+		},
+		"Node.js": {
+			"url": "https://nodejs.org/",
+			"logo": "/images/nodejs.svg",
```

Comment:
Looking at local output, this logo seems out of line compared to others, and details are not very readable at this size anyway:
![image](https://user-images.githubusercontent.com/557590/85141517-cb7e7400-b23e-11ea-8adc-0209c24147d3.png)

Could we use the icon version instead? It seems to be allowed as a "primary icon + pattern" in [official visual guidelines](https://nodejs.org/static/documents/foundation-visual-guidelines.pdf) and would look better at this size.

I don't know where to get SVG version for it, but then, I'm not sure where you got this one either TBH 😀

Reply:
I got it from wikipedia, https://en.wikipedia.org/wiki/Node.js#/media/File:Node.js_logo.svg

I don't see an icon version there or on https://nodejs.org/en/about/resources/

Doing some more searching, I can't find a download for the "primary icon + pattern" anywhere... so I used Inkscape to crop the existing logo...

Reply:
I found one in their page source but it seems to be a sprite image and would need to be cropped or manually extracted too.

**UPD**: actually, looks like that sprite is built from separate sources, and the one we want is here: https://github.com/nodejs/nodejs.org/blob/master/static/images/logo-hexagon.svg

Reply:
I'm fine with keeping either of those, just need to check if SVGs are optimised as part of the build or we need to compress the current version with SVGO...

Reply:
Looks like we don't; can you please either compress this SVG or add the one from Node repo (which looks already pre-compressed)?

Reply:
Sure, I used the one from that link (since I have no idea how to compress one myself...)

Reply:
Oh, just in case you ever need that in the future - as I mentioned above, there is SVGO (probably the most popular SVG optimiser out there), and you can use it either via CLI, or, for example, via this Web UI by one of our fellow Web DevRels :) https://jakearchibald.github.io/svgomg/

Reply:
Thanks for updating!